src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy

```diff
@@ -161,24 +169,23 @@ class PartialUpdateSpec extends HealthCheckSpecification {
     def "Able to partially update flow #data.field which causes a reroute"() {
         given: "A flow"
         def swPair = switchPairs.all().random()
-        def flow = flowHelperV2.randomFlow(swPair)
-        flowHelperV2.addFlow(flow)
-        def originalCookies = northbound.getSwitchRules(swPair.src.dpId).flowEntries.findAll { !new Cookie(it.cookie).serviceFlag }
+        def flow = flowFactory.getRandom(swPair)
+        def originalCookies = switchRulesFactory.get(swPair.src.dpId).getRules().findAll { !new Cookie(it.cookie).serviceFlag }
 
         when: "Request a flow partial update for a #data.field field"
         def newValue = data.getNewValue(flow."$data.field")
         def updateRequest = new FlowPatchV2().tap { it."$data.field" = newValue }
-        def response = flowHelperV2.partialUpdate(flow.flowId, updateRequest)
+        def response = flow.partialUpdate(updateRequest)
```

Comment:
flow.partialUpdate(updateRequest) includes waiting for flow to be in UP state
According to the test, we need to check response from update operation(smth like sendCreateRequest()) and then check flow details.

Reply:
Done